dist/pom.xml

```diff
@@ -24,7 +24,7 @@
         <artifactId>rapids-4-spark-parent</artifactId>
         <version>22.12.0-SNAPSHOT</version>
     </parent>
-    <artifactId>rapids-4-spark_2.12</artifactId>
+    <artifactId>rapids-4-spark-${cpu_arch}_2.12</artifactId>
```

Comment:
Based on an earlier discussion, I thought it was not allowed to have a variable in the artifactId?  Seems like we want to split dist into dist-amd64 and dist-arm64, and maybe they rely on a dist-common that builds an intermediate jar using the CPU architecture in the classifier, similar to how we use the Spark version in the classifier of our sql-plugin intermediate jars.

Reply:
> I thought it was not allowed to have a variable in the artifactId?

maven does not allow to put a var in project root module (rapids-4-spark-parent in our case), 
we could add it in submodule artifactID.